fiftyone/brain/__init__.py

```diff
@@ -712,103 +712,98 @@ def compute_exact_duplicates(
 
 def compute_leaky_splits(
     samples,
-    brain_key=None,
-    split_views=None,
-    split_field=None,
     split_tags=None,
+    split_field=None,
+    split_views=None,
     threshold=0.2,
-    similarity_brain_key=None,
```

Comment:
These changes:
- `embeddings_field` -> `embeddings`
- `similarity_brain_key` -> `similarity_index`

are for consistency with other methods like `compute_uniqueness()`.

The idea behind removing `similarity_backend` and `similarity_config_dict`, aside from also being consistent with other brain methods, is basically: if you want fine-grained control over the similarity index, just make one yourself and pass it in via `similarity_index`.